scala.html.markdown

```diff
@@ -276,6 +276,7 @@ r foreach println
 // NB: Scala is quite lenient when it comes to dots and brackets - study the
 // rules separately. This helps write DSLs and APIs that read like English
 
+// Why doesn't `println` need any parameters here? Stay tuned for Functional Programming below!
```

Comment:
I just meant "functional programming" -> "first class functions" ("stay tuned for first class functions")

the term "functional programming" is (I believe) a bit too overloaded to be used meaningfully